### ISSUE TITLE
The XML element selah is not valid OSIS

### ISSUE THREAD

**Issue by DavidHaslam** [adyeths/u2o#58]
`u2o.py` converts `\qs ...\qs*` to `<selah>...</selah>` but this XML element is not valid OSIS.

It should convert it to an **l** element with **type** selah. i.e. `<l type="selah">...</l>`
```
A particular use of the l element is for marking the Hebrew term "Selah" which occurs in many Psalms.
"Selah" (however it is translated or transliterated) should be marked up like this:
<l type="selah">Selah</l>
The l element's type attribute can have the following values:
• attribution
• doxology
• refrain
• selah
As with other attribute values, values beginning with "x-" are also permitted.
```
From section **11.5.2** of the OSIS 2.1 User Reference (page 65).

**Comment by DavidHaslam** [comment]
These code lines in `u2o.py` are where the element is made:
```
# other introduction and poetry tags
OTHERTAGS = OrderedDict()
for _ in [
        # selah is handled in a special manner…
        (r'\qs ', '<selah>'),
        (r'\qs*', '</selah>'),
        # these tags get special handling…
        (r'\ie', '<!-- ie -->'),  # handled with partags… may not need here…
        (r'\ib ', '<!-- b -->'),  # handled exactly like \b
        (r'\b ', '<!-- b -->'),
        (r'\nb ', '<!-- nb -->'),
        (r'\nb', '<!-- nb -->'),
        # translators chunk marker…
        (r'\ts', '<!-- ts -->')]:
    OTHERTAGS[_[0]] = _[1]
```

NB. There's a procedure called **fixselah** that's supposed to then fix the `<selah>...</selah>` in the text.
For some unknown cause, this was not being processed.

**Comment by DavidHaslam** [comment]
Here's a snippet of my input USFM file (from Habakkuk 3):
```
\v 3 Tamy avy any Temana Andriamanitra, Dia ny iray Masina avy any an-tendrombohitra Parana. \qs (Sela.)\qs* Manarona ny lanitra ny voninahiny, Ary manenika ny tany ny fiderana Azy.

```
Here's the corresponding OSIS output:
```
<verse sID="Hab.3.3" osisID="Hab.3.3" n="3" />Tamy avy any Temana Andriamanitra, Dia ny iray Masina avy any an-tendrombohitra Parana. <selah>(Sela.)</selah> Manarona ny lanitra ny voninahiny, Ary manenika ny tany ny fiderana Azy.
<verse eID="Hab.3.3" />
```

**Comment by DavidHaslam** [comment]
Yep! That's the root cause. 

After I inserted `\q ` before each of the three verses, the output became (e.g.):
```
<lg>
<l level="1">
<verse sID="Hab.3.3" osisID="Hab.3.3" n="3" />Tamy avy any Temana Andriamanitra, Dia ny iray Masina avy any an-tendrombohitra Parana.</l><l type="selah">(Sela.)</l><l> Manarona ny lanitra ny voninahiny, Ary manenika ny tany ny fiderana Azy.
<verse eID="Hab.3.3" />
...
```

**Comment by DavidHaslam** [comment]
That's what I concluded after my simple test. 

I'm not using **ParaTExt** to create my USFM files; just relying upon my understanding of the USFM documentation. I don't even know whether **ParaTExt** would reject `\qs ...\qs*` with an error message or warning should it encounter these markers outside poetry lines.

Aside: My fall-back position for the ongoing project is simply to disable the tagging of `(Sela.)`.
It's of minor consequence, given that these items aren't even present in the Psalms for this old translation.

**Comment by adyeths** [comment]
This should be fixed now. Check it to make sure.

**Comment by DavidHaslam** [comment]
Please fix issue #62 so that I can check this solution.

**Comment by adyeths** [comment]
The issue with #62 is unrelated to this one. I will fix that issue in the near future.
